serum-390/soen390-team16-godzilla
# Packaging

# Description

Added an API route, handler and made changes to the database to incorporate packing products

## Link to work item

api/packagedproduct/

## How to Test

1- Spin up docker
2- mvn spring-boot:run
3- Check api/packagedproducts/
4- Send GET/PUT/POST/DELETE requests on postman
